### ISSUE TITLE
Kernal died while importing torchvision

### ISSUE THREAD

**Issue by tiwari93** [pytorch/vision#5953]
### 🐛 Describe the bug

Unable to import torchvision. Kernal appears to have died

Code used in jupyter notebook

Import torch
import torchvision

I tested creating python script and running from command prompt same issue.

python version - 3.9.12
toch version - 1.11.0
torchvision - 0.12.0
 windows 10

I tried the same on other version of python 3.7 with compatible version of torch same issue.
Earlier everything was working fine. Facing this issue from past 4 days

![image](https://user-images.githubusercontent.com/49356997/166922316-ec1c68ac-a149-4378-8f7d-8dcc13e4315b.png)

### Versions

![image](https://user-images.githubusercontent.com/49356997/166922369-86a4fccd-0eb6-4b6c-9db9-cf71d01ffbed.png)

**Comment by NicolasHug** [comment]
Hi @tiwari93 , do you mind reproducing the code outside of a notebook so that we can see the actual error message?
Also please run and report

```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

**Comment by tiwari93** [comment]
> 
Hi @NicolasHug,
I tried running the same code on command prompt but it simply came out of python mode:

![image](https://user-images.githubusercontent.com/49356997/167058940-74a18a6d-9673-41d0-9117-e4aa572eb475.png)

Also pls find the env.py file report:

Collecting environment information...
PyTorch version: 1.11.0+cpu
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 10 Pro
GCC version: (x86_64-posix-seh, Built by strawberryperl.com project) 8.3.0
Clang version: Could not collect
CMake version: version 3.17.2
Libc version: N/A

Python version: 3.9.12 (tags/v3.9.12:b28265d, Mar 23 2022, 23:52:46) [MSC v.1929 64 bit (AMD64)] (64-bit runtime)
Python platform: Windows-10-10.0.19044-SP0
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] Could not collect
[conda] Could not collect

**Comment by pmeier** [comment]
Something seems off with your env. The script found PyTorch, but didn't find `torchvision`. How did you install it?

**Comment by tiwari93** [comment]
I used pip install torchvision
Not sure what is the issue earlier it was working perfectly.

**Comment by pmeier** [comment]
What does `pip show torchvision` give you?

**Comment by pmeier** [comment]
While we are at it, please also post the output of

```shell
$ pip -V
$ python -m pip -V
```

**Comment by tiwari93** [comment]
> pip list --format=freeze

absl-py==1.0.0
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.5
astunparse==1.6.3
attrs==21.4.0
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.0
cachetools==5.0.0
certifi==2021.10.8
cffi==1.15.0
charset-normalizer==2.0.12
colorama==0.4.4
cycler==0.11.0
debugpy==1.6.0
decorator==5.1.1
defusedxml==0.7.1
entrypoints==0.4
executing==0.8.3
fastjsonschema==2.15.3
flatbuffers==2.0
fonttools==4.33.3
gast==0.5.3
google-auth==2.6.6
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.46.0
h5py==3.6.0
idna==3.3
imageio==2.19.0
importlib-metadata==4.11.3
ipykernel==6.13.0
ipython==8.3.0
ipython-genutils==0.2.0
jedi==0.18.1
Jinja2==3.1.2
joblib==1.1.0
jsonschema==4.4.0
jupyter-client==7.3.0
jupyter-core==4.10.0
jupyterlab-pygments==0.2.2
keras==2.8.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.2
libclang==14.0.1
Markdown==3.3.6
MarkupSafe==2.1.1
matplotlib==3.5.2
matplotlib-inline==0.1.3
mistune==0.8.4
nbclient==0.6.2
nbconvert==6.5.0
nbformat==5.4.0
nest-asyncio==1.5.5
networkx==2.8
notebook==6.4.11
numpy==1.22.3
oauthlib==3.2.0
opencv-python==4.5.5.64
opt-einsum==3.3.0
packaging==21.3
pandas==1.4.2
pandocfilters==1.5.0
parso==0.8.3
pickleshare==0.7.5
Pillow==9.1.0
pip==22.0.4
prometheus-client==0.14.1
prompt-toolkit==3.0.29
protobuf==3.20.1
psutil==5.9.0
pure-eval==0.2.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
Pygments==2.12.0
pyparsing==3.0.8
pyrsistent==0.18.1
python-dateutil==2.8.2
pytz==2022.1
PyWavelets==1.3.0
pywin32==304
pywinpty==2.0.5
pyzmq==22.3.0
requests==2.27.1
requests-oauthlib==1.3.1
rsa==4.8
scikit-image==0.19.2
scikit-learn==1.0.2
scipy==1.8.0
Send2Trash==1.8.0
setuptools==58.1.0
six==1.16.0
soupsieve==2.3.2.post1
stack-data==0.2.0
tensorboard==2.8.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.8.0
tensorflow-io-gcs-filesystem==0.25.0
termcolor==1.1.0
terminado==0.13.3
tf-estimator-nightly==2.8.0.dev2021122109
threadpoolctl==3.1.0
tifffile==2022.5.4
tinycss2==1.1.1
torch==1.11.0
torchaudio==0.11.0
torchvision==0.12.0
tornado==6.1
traitlets==5.1.1
typing_extensions==4.2.0
urllib3==1.26.9
wcwidth==0.2.5
webencodings==0.5.1
Werkzeug==2.1.2
wheel==0.37.1
wrapt==1.14.1
zipp==3.8.0

**Comment by tiwari93** [comment]
> ```shell
> pip -V
> ```

response of pip -v
pip 22.0.4 from C:\Program Files\Python39\lib\site-packages\pip (python 3.9)


response of python -m pip -V
pip 22.0.4 from C:\Program Files\Python39\lib\site-packages\pip (python 3.9)

**Comment by pmeier** [comment]
Hm, nothing obvious stands out in the env. Still, it is quite heavy. As a last step, could you try to reproduce this in a clean env?

```
$ pip install virtualenv
$ virtualenv venv
$ .\venv\Scripts\activate
$ pip install torch torchvision
$ python -c 'import torch; import torchvision'
```

**Comment by tiwari93** [comment]
> Hm, nothing obvious stands out in the env. Still, it is quite heavy. As a last step, could you try to reproduce this in a clean env?
> 
> ```
> $ pip install virtualenv
> $ virtualenv venv
> $ .\venv\Scripts\activate
> $ pip install torch torchvision
> $ python -c 'import torch; import torchvision'
> ```

Even on virtual environment facing the same issue:

![image](https://user-images.githubusercontent.com/49356997/167619179-5c5dc95c-7a06-48d1-9a8b-2bd87743d295.png)

**Comment by pmeier** [comment]
Good thing that worked for you. I couldn't reproduce in a fresh Windows VM so this would have gotten really hard to pinpoint.

**Comment by tiwari93** [comment]
> Good thing that worked for you. I couldn't reproduce in a fresh Windows VM so this would have gotten really hard to pinpoint.

Thank you all for your support on this
